### ISSUE TITLE
[BUG] Model is overfitting

### ISSUE THREAD

**Issue by SupurCalvinHiggins** [thourihan/CSC461_Final#2]
The model appears to be overfitting. Validation accuracy is ~96% but test accuracy is only ~83%. 

I propose the following potential fixes:
1. Add weight decay.
2. Add regularization.
3. Add/increase dropout.
